src/dataProvider.js

```diff
@@ -15,35 +15,35 @@ function graphQlRequest(query, variables, options) {
             variables: variables,
         })
     })
-    .then(r => r.json())
-    .then(response => {
-        console.log('data returned:', response); 
-        return response.data;
-    });
+        .then(r => r.json())
+        .then(response => {
+            console.log('data returned:', response);
+            return response.data;
+        });
 }
 
 const providers = {
-    getUser: function() {
+    getUser: function () {
         return graphQlRequest('{ user { name, roles } }', null, { cookies: true });
     },
 
-    getTeamsStandings: function() {
+    getTeamsStandings: function () {
```

Comment:
did you made this formatting changes on intention or do you have any plugin that does this? In either case, please dont do this. Only change lines where you are making logic changes

Reply:
I didn't notice .... maybe because I use Ctrl+A , then Ctrl+K+F to align the code in VS Code

Reply:
maybe, don do that please ;) ... I will setup linter that will auto format all our code, just didnt have time yet